apps/walletd/dialogs/WalletSendLedgerDialog/index.tsx

```diff
@@ -28,91 +23,30 @@ export function WalletSendLedgerDialog({
   open,
   onOpenChange,
 }: Props) {
-  const { walletId } = dialogParams || {}
-  const [step, setStep] = useState<SendStep>('compose')
-  const [signedTxnId, setSignedTxnId] = useState<string>()
-  const [sendParams, setSendParams] = useState<SendParamsV1>(emptySendParamsV1)
-  const balance = useWalletBalance({
-    disabled: !walletId,
-    params: {
-      id: walletId,
-    },
-  })
-  const { dataset: addresses } = useWalletAddresses({ id: walletId })
-
-  const balanceSc = useMemo(
-    () => new BigNumber(balance.data?.siacoins || 0),
-    [balance.data]
-  )
+  const n = useConsensusNetwork()
+  const ct = useConsensusTip()
 
-  const balanceSf = useMemo(
-    () => new BigNumber(balance.data?.siafunds || 0),
-    [balance.data]
-  )
-
-  // Form for each step
-  const compose = useComposeFormV1({
-    balanceSc,
-    balanceSf,
-    defaultChangeAddress: addresses?.[0]?.address,
-    defaultClaimAddress: addresses?.[0]?.address,
-    onComplete: (data) => {
-      setSendParams((d) => ({
-        ...d,
-        ...data,
-      }))
-      setStep('send')
-    },
-  })
-  const send = useSendForm({
-    walletId,
-    step,
-    params: sendParams,
-    onConfirm: ({ transactionId }) => {
-      setSignedTxnId(transactionId)
-      setStep('done')
-    },
+  const version = getCurrentVersion({
```

Comment:
Actually, should we continue V1 txns until the require height and then show the disabled dialog v2 warning?

Reply:
Adding in follow up PR.